src/databricks/labs/ucx/hive_metastore/pipelines_migrate.py

```diff
@@ -67,9 +69,28 @@ def _populate_pipeline_job_tasks_mapping(self) -> None:
 
     def _get_pipelines_to_migrate(self) -> list[PipelineInfo]:
```

Comment:
IMO `_get_pipelines_to_migrate` should be merged with `_list_pipelines` as it is now unclear which method to use in other methods